python/dask_cudf/pyproject.toml

```diff
@@ -36,6 +36,7 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
 ]
+dynamic = ["entry_points"]
```

Comment:
Getting a new error locally:
```
writing list of installed files to 'record.txt'
/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py:108: _BetaConfiguration: Support for `[tool.setuptools]` in `pyproject.toml` is still *beta*.
  warnings.warn(msg, _BetaConfiguration)
configuration error: `project.dynamic[0]` must be one of ['version', 'description', 'readme', 'requires-python', 'license', 'authors', 'maintainers', 'keywords', 'classifiers', 'urls', 'scripts', 'gui-scripts', 'entry-points', 'dependencies', 'optional-dependencies']
GIVEN VALUE:
    "entry_points"

OFFENDING RULE: 'enum'

DEFINITION:
    {
        "enum": [
            "version",
            "description",
            "readme",
            "requires-python",
            "license",
            "authors",
            "maintainers",
            "keywords",
            "classifiers",
            "urls",
            "scripts",
            "gui-scripts",
            "entry-points",
            "dependencies",
            "optional-dependencies"
        ]
    }
Traceback (most recent call last):
  File "/nvme/0/pgali/cudf/python/dask_cudf/setup.py", line 5, in <module>
    setup(
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/__init__.py", line 108, in setup
    return distutils.core.setup(**attrs)
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 159, in setup
    dist.parse_config_files()
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/dist.py", line 885, in parse_config_files
    pyprojecttoml.apply_configuration(self, filename, ignore_option_errors)
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 62, in apply_configuration
    config = read_configuration(filepath, True, ignore_option_errors, dist)
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 126, in read_configuration
    validate(subset, filepath)
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 51, in validate
    raise ValueError(f"{error}\n{summary}") from None
ValueError: invalid pyproject.toml config: `project.dynamic[0]`.
configuration error: `project.dynamic[0]` must be one of ['version', 'description', 'readme', 'requires-python', 'license', 'authors', 'maintainers', 'keywords', 'classifiers', 'urls', 'scripts', 'gui-scripts', 'entry-points', 'dependencies', 'optional-dependencies']
(cudfdev) pgali@dt07:/nvme/0/pgali/cudf$ 
```

Reply:
Whoops that was a typo on my part. I had `entry_points` instead `entry-points`.